docs/Parallel-Building.md

```diff
@@ -25,9 +25,8 @@ These instructions assume you use Homebrew, and have installed Theos as we recom
     echo PATH=\"$(brew --prefix make)/libexec/gnubin:\$PATH\" >> ~/.zprofile
     ```
 
-    For versions of macOS earlier than 10.15 Catalina, the default shell is bash rather than zsh. Use `~/.bash_profile` instead of `~/.zprofile`.
-3. Close this terminal tab, and create a new one, in order for the profile script change to take effect.
-4. Test the `make` command. The notice should no longer appear, and you should find that it’s significantly faster!
+    For versions of macOS earlier than 10.15 Catalina, the default shell is bash rather than zsh. As such, you'll want to use `~/.bash_profile` instead of `~/.zprofile` in the command above.
```

Comment:
Extremely annoying nitpick: use smart (curly) quotes. `you'll` → `you’ll`. If on macOS you can type them with opt-] (opening apostrophe) and opt-shift-] (closing apostrophe).